torch/linalg/__init__.py

```diff
@@ -1813,6 +1822,83 @@
     https://en.wikipedia.org/wiki/Invertible_matrix#The_invertible_matrix_theorem
 """)
 
+solve_triangular = _add_docstr(_linalg.linalg_solve_triangular, r"""
+linalg.solve_triangular(A, B, *, upper, left=True, unitriangular=False, out=None) -> Tensor
```

Comment:
I appreciate making upper required but we should review being consistent with this approach vs the compatibility cost.

We don't need to do that review before landing this PR, however. Changing upper to have a default later should be OK.